notebooks/ch12/linear_case_study.md

```diff
@@ -0,0 +1,1205 @@
+      <th></th>
+      <th>Length</th>
+      <th>Girth</th>
+      <th>Height</th>
+      <th>...</th>
+      <th>Age_2-5</th>
+      <th>Age_&lt;2</th>
+      <th>Age_&gt;20</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>465</th>
+      <td>98</td>
+      <td>113</td>
+      <td>99</td>
+      <td>...</td>
+      <td>1</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>233</th>
+      <td>101</td>
+      <td>119</td>
+      <td>101</td>
+      <td>...</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>450</th>
+      <td>106</td>
+      <td>125</td>
+      <td>103</td>
+      <td>...</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>453</th>
+      <td>93</td>
+      <td>120</td>
+      <td>100</td>
+      <td>...</td>
+      <td>1</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>452</th>
+      <td>98</td>
+      <td>120</td>
+      <td>108</td>
+      <td>...</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+  </tbody>
+</table>
+<p>5 rows × 13 columns</p>
+</div>
+
+
+
+We should also add a column of biases in order to have a constant term in our model.
+
+
+```python
+donkeys_train = donkeys_train.assign(bias=1)
+```
+
+
+```python
+# HIDDEN
+donkeys_train = donkeys_train.reindex_axis(['bias'] + list(donkeys_train.columns[:-1]), axis=1)
+
+```
+
+
+```python
+donkeys_train.head()
+```
+
+
+
+
+<div>
+<style>
+    .dataframe thead tr:only-child th {
+        text-align: right;
+    }
+
+    .dataframe thead th {
+        text-align: left;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>bias</th>
+      <th>Length</th>
+      <th>Girth</th>
+      <th>...</th>
+      <th>Age_2-5</th>
+      <th>Age_&lt;2</th>
+      <th>Age_&gt;20</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>465</th>
+      <td>1</td>
+      <td>98</td>
+      <td>113</td>
+      <td>...</td>
+      <td>1</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>233</th>
+      <td>1</td>
+      <td>101</td>
+      <td>119</td>
+      <td>...</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>450</th>
+      <td>1</td>
+      <td>106</td>
+      <td>125</td>
+      <td>...</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>453</th>
+      <td>1</td>
+      <td>93</td>
+      <td>120</td>
+      <td>...</td>
+      <td>1</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>452</th>
+      <td>1</td>
+      <td>98</td>
+      <td>120</td>
+      <td>...</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+  </tbody>
+</table>
+<p>5 rows × 14 columns</p>
+</div>
+
+
+
+## Multiple Linear Regression Model
```

Comment:
Overall I think we want to be more precise about this progression from one variable to multiple variables